packaging_automation/update_package_properties.py

```diff
@@ -40,22 +66,36 @@ class PackagePropertiesParams:
     latest_changelog: str = ""
     changelog_date: datetime = datetime.now()
 
+    def changelog_add(self) -> bool:
+        return not self.fancy or self.fancy_version_number == 1
+
     def spec_file_name(self) -> str:
-        return spec_file_name(self.project_name)
+        return spec_file_name(self.supported_project.value.name)
+
+    def pkgvars_template_file_name(self) -> str:
+        return f"{self.supported_project.value.name}-pkgvars.tmpl"
+
+    def rpm_spec_template_file_name(self) -> str:
+        return f"{self.supported_project.value.name}.spec.tmpl"
```

Comment:
I think those three could be `@property` along with many functions of this class, but we can improve that in another pr too.

Reply:
can you open an issue for that ?

Reply:
Actually I couldn't understand can you give me a little bit detail?

Reply:
In python, it is pretty usual to use `@property` decorator for the member functions that are doing simple calculations that are not modifying object state but returning a "property" of given object without storing that property as a member variable.

I think we should use that decorator for the member functions that are evaluating such strings etc.

https://www.machinelearningplus.com/python/python-property/
(especially the second section)

Reply:
But we can do that in another pr and open an issue for that for now.

Reply:
Fixed